release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1625,6 +1625,8 @@ ERROR failed to fetch Master Machines: failed to load asset "Install Config": fa
 +
 As a workaround, if you have an installation configuration file, update it with a specific project id to use (`platform.gcp.projectID`). Otherwise, manually create an installation configuration file, and enter a specific project id. Run the installation program again, specifying the file. (link:https://issues.redhat.com/browse/OCPBUGS-15238[*OCPBUGS-15238*])
 
+* Booting fails in a large compute node. (link:https://issues.redhat.com/browse/RHEL-11776[*RHEL-11776*])
```

Comment:
Not sure if just the bug number is good enough here, the intention for that bug to find the cause and fix it.

Reply:
@mkumatag updated the PR with the https://issues.redhat.com/browse/OCPBUGS-20075 link. Request you to check. Thanks.

Reply:
@mkumatag could you please confirm if you agree with above update ? Thanks.